main.go

```diff
@@ -17,9 +18,9 @@ import (
 	"github.com/urfave/cli"
 )
 
-// version must be set from the contents of VERSION file by go build's
-// -X main.version= option in the Makefile.
-var version = "unknown"
+// version is automatically set from the contents of VERSION file on go1.16+.
+//go:embed VERSION
+var version string
```

Comment:
Is it possible to make it const?

Reply:
No it's not :( Not a big deal I guess.

Reply:
Unfortunately not possible indeed (I also went looking for that 😂)